augraphy/utilities/function.py

```diff
@@ -6,25 +6,22 @@ class Function(Augmentation):
     """Accepts an arbitrary function or list of functions to apply in the pipeline.
 
     :param fs: The function(s) to apply.
-    :type fs: function or list of functions
```

Comment:
I think this can be simpler and clearer with just `AugmentationSequence` wrapping multiple `Function`? With this it will be more code efficient and less redundancy. It shouldn't be a problem if we want to revert it back to support list too.

Reply:
As long as supporting the function list here doesn't conflict, then leaving it is similar to the idea of having the `__call__` interface in that is provides ways to do things that require less typing.  Fewer things to read can also increase readability as long as the intent remains clear.

Reply:
Alright, then i will include it in the next update.